app/services/dqt/retrieve_teacher.rb

```diff
@@ -9,7 +9,7 @@ def initialize(trainee:)
     end
 
     def call
-      Client.get("/v2/teachers/#{trainee.trn}?birthdate=#{trainee.date_of_birth}")
+      Client.get("/v2/teachers/#{trainee.trn}?birthdate=#{trainee.date_of_birth}&includeInactive=true")
```

Comment:
The other place this service is used is in `Dqt::RetrieveTraining` - will this affect the logic in there? In `Dqt::RetrieveTraining` we're picking out the correct training based on the UKPRN and the training route. Now that we're including inactive trainings, is it possible we'll match on two or more?

Reply:
I guess that would only be the case if a trainee had withdrawn from a course with a provider, and then started again with the same provider.

Reply:
idea: in Dqt::RetrieveTraining could we raise an error when two possible trainings are identified? at the moment it just does a find so would pick the first one.